### PR TITLE
Creating Contact form

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello dear !
I'm openning a new pull request to address the changes I just made in the project by adding a contact form that collects user datas and send me An email when the form is submitted.
Please let me know if there's anything I can improve or if the work has been perfectly done !
Yours faithfully 🤝  😀

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -279,7 +279,18 @@ <h3 class="languages-title">Skills</h3>
                 </li>
 
             </section>
-
+            <section class="contact-section">
+                <h4 class="contact">Contact me</h4>
+                <p class="contact-text">If you have an application you are interested in developing, a feature that you
+                    need built or a project that needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/mbjbpybo" method="post" class="contact-form">
+                    <input placeholder="name" class="name" max="30" name="name" type="text" required>
```
Comment (topeogunleye) [inline index.html]: - [ ] I kindly suggest that you change max to maxlength because it is required by the project.

File: index.html [hunk 2]
```diff
@@ -279,7 +279,18 @@ <h3 class="languages-title">Skills</h3>
...
+                    <input placeholder="email" class="email" name="email" type="email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" maxlength="500" class="text-box"
```
Comment (topeogunleye) [inline index.html:289]: - [ ] As required, please kindly ensure that all input fields have the `required` attribute to activate the validation on submit.

File: styles.css [hunk 3]
```diff
@@ -428,3 +432,110 @@ header {
...
+  margin-right: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  align-items: center;
+  color: #172b4d;
+  padding: 10px;
+}
+
+.button-contact {
```
Comment (topeogunleye) [inline styles.css:524]: - [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed) as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @chiztechnology and Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
thanks alot for all your reviews !
I've just applied them in the projects.
I really apprecite 🤝

**Review by ichala — APPROVED** [review]
Hi @chiztechnology ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> thanks alot for all your reviews ! I've just applied them in the projects. I really apprecite 🤝

@chiztechnology you're welcome

**Pull request merged**